ResourceStatusSystem/Agent/SiteInspectorAgent.py

```diff
@@ -133,16 +135,39 @@ def getSitesToBeChecked(self):
     if not res['OK']:
       return res
 
+    utcnow = datetime.datetime.utcnow().replace(microsecond=0)
+
     # filter elements
     for site in res['Value']:
       status = res['Value'].get(site, 'Unknown')
 
+      # Maybe an overkill, but this way I have NEVER again to worry about order
+      # of elements returned by mySQL on tuples
+      siteDict = dict(zip(res['Columns'], site))
```

Comment:
I am not sure I understand the comment. Dict are not ordered

Reply:
Yes, but the content of the tuples returned by the call to MySQL could be.